hello-world/src/main/resources/archetype-resources/src/main/resources/HelloWorldBuilder/help-useFrench_es.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    \u00BFUtilizar el franc\u00E9s?
```

Comment:
You cannot use `\u` escapes in HTML. Treat as UTF-8 and use the raw characters, or use HTML entities.

Reply:
Acc. to [this code](https://github.com/jenkinsci/jenkins/blob/8189c2cd9a2cb0a4e6d2dcf341fb818dbd9165ba/core/src/main/java/hudson/model/Descriptor.java#L940-L942) it is treated as UTF-8 regardless of master’s encoding, so go ahead and use raw UTF-8.